src/simperium/channel.js

```diff
@@ -137,6 +137,15 @@ internal.findAcknowledgedChange = function( change ) {
 	}
 };
 
+internal.requestObjectVersion = function( id, version ) {
+	return new Promise( resolve => {
+		this.once( `version.${ id }.${ version }`, data => {
+			resolve( data );
+		} );
```

Comment:
a bit verbose?

```js
this.once( `version.${ id }.${ version }`, resolve )
```

those are identical other than the fact that this version actually returns the value of `resolve( data )` but that's somewhat irrelevant since we're not grabbing the return value.